app/dturn/README.md

```diff
@@ -27,47 +27,48 @@ const {
 
 ### initDiscussion( discussionId, options )
 
--   **discussionId** is a globally unique string to identify the discussion. Like a Mongo ObjectId
--   **options** is an object of properties to initialize the discussion with
-    -   **group_size**: the size of a group, like 10
-    -   **gmajority**: the multiplyer to calculate the majority size, like 0.5
-    -   **max_rounds**: maximum depth to collect statments in grouping, like 10
-    -   **min_shown_count**:the minimum number of times a item pair is shown in order to decide if a majority have grouped it like Math.floor(group_size/2)+1
-    -   **min_rank**: when filterning statements for the next round, they must at least have this number of users voting for it
-    -   **updateUInfo**: function to call to write out incremental changes to a database for later rehydration
-    -   **getAllUInfo**: funtion to get all the incremental changes from a database to rehydrate the deliberation
--   **returns** a promse that is fulfilled when the initialition is complete, includeing the call to getAllUinfo
+- **discussionId** is a globally unique string to identify the discussion. Like a Mongo ObjectId
+- **options** is an object of properties to initialize the discussion with
+  - **group_size**: the size of a group, like 10
+  - **gmajority**: the multiplyer to calculate the majority size, like 0.5
+  - **max_rounds**: maximum depth to collect statments in grouping, like 10
+  - **min_shown_count**:the minimum number of times a item pair is shown in order to decide if a majority have grouped it like Math.floor(group_size/2)+1
+  - **min_rank**: when filterning statements for the next round, they must at least have this number of users voting for it
+  - **updateUInfo**: function to call to write out incremental changes to a database for later rehydration
+  - **getAllUInfo**: funtion to get all the incremental changes from a database to rehydrate the deliberation
+  - **updates**: function to send updates to all connected clients. Not to be confused with updateUInfo.
```

Comment:
great, just add this to the end of the line: `{participants: number, lastRound: number}`